testdata/README.md

```diff
@@ -7,6 +7,8 @@ http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/
 
 The variants are in genome build hg19.
 
+Test data for king.wdl is about 149.9 Mb and available as ALL.pruned.gds in: gs://topmed_workflow_testing/UWGAC_WDL/ALL.pruned.gds
```

Comment:
You may want to change this to the storage.google.cloud link in the README, as that is the form that internet browsers can resolve.

ie, https://storage.google.com/topmed_workflow_testing/UWGAC_WDL/ALL.pruned.gds

Reply:
Resolved in the last commit.